apps/spruce/src/constants/requesters.ts

```diff
@@ -11,18 +11,16 @@ enum Requester {
 }
 
 const requesterToTitle: PartialRecord<Requester, string> = {
-  [Requester.AdHoc]: "Ad Hoc Request",
+  [Requester.AdHoc]: "Periodic build",
   [Requester.GitHubMergeQueue]: "GitHub Merge Request",
   [Requester.GitHubPR]: "GitHub Pull Request",
-  [Requester.GitTag]: "Git Tag Request",
-  [Requester.Gitter]: "Gitter Request",
-  [Requester.Patch]: "Patch Request",
-  [Requester.Trigger]: "Trigger Request",
+  [Requester.GitTag]: "Git tag",
+  [Requester.Gitter]: "Commit",
+  [Requester.Patch]: "Patch",
```

Comment:
nit. Could we use a consistent case here? (choose one between sentence case and title case)